### PR TITLE
feat(calendar/header): add disableArrowLeft/disableArrowRight props

### PR DESCRIPTION
**Pull request by lfkwtz**
Add `disableArrowLeft` and `disableArrowRight` props to Calendar header component. Also added a `disabledArrowColor` prop to the theme object

### CODE REVIEW

**Comment by tautvilas** [comment]
Wouldn't it be better just to user renderArrow to implement custom logic for disabling? I'm afraid this will make API too complicated

**Comment by lfkwtz** [comment]
I see it the other way personally - I see disabling an arrow as a baseline feature, and creating a custom component to conditionally trigger that logic seems like overkill.

File: src/calendar/header/index.js [hunk 1]
```diff
@@ -63,16 +65,16 @@ class CalendarHeader extends Component {
   }
 
   onPressLeft() {
-    const {onPressArrowLeft} = this.props;
-    if(typeof onPressArrowLeft === 'function') {
+    const { onPressArrowLeft } = this.props;
```
Comment (Inbal-Tish) [inline src/calendar/header/index.js]: Please remove all formatting changes

**Comment by Inbal-Tish** [comment]
@lfkwtz I agree that disabling the arrows should be a feature. Please remove all formatting changes so I can review the relevant ones and please make sure the disable functionality can dynamically be updated. Also, resolve conflict, please. Thanks

**Comment by lfkwtz** [comment]
cal

@Inbal-Tish should be good to go now

**Pull request merged**